Simplenote/Classes/CSSearchable+Helpers.swift

```diff
@@ -32,8 +32,29 @@ extension CSSearchableItem {
 }
 
 extension CSSearchableIndex {
+    // MARK: - Index Notes
+    @objc
+    func indexSpotlightItems(in context: NSManagedObjectContext) {
+        guard Options.shared.indexNotesInSpotlight else {
+            return
+        }
+
+        context.perform {
+            if let deleted = context.fetchObjects(forEntityName: "Note", with: NSPredicate(format: "deleted == YES")) as? [Note] {
+                CSSearchableIndex.default().deleteSearchableNotes(deleted)
+            }
+
+            if let notes = context.fetchObjects(forEntityName: "Note", with: NSPredicate(format: "deleted == NO")) as? [Note] {
+                CSSearchableIndex.default().indexSearchableNotes(notes)
```

Comment:
I think you can replace `CSSearchableIndex.default()` with `self` ? (same note a couple lines above!!)